app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-core.js

```diff
@@ -214,18 +214,18 @@ window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {};
     },
 
     ecommerceHelperFunctions: {
-      clearEcommerceObject: function() {
+      clearEcommerceObject: function () {
```

Comment:
This commit is called 'Add tests' but most of the changes in this file look like linting tweaks. Probably the easiest thing would be to merge this commit with the related earlier commit, so the linting tweaks are included and the tests accompany the code they relate to.